tools/ci/cibuild.sh

```diff
@@ -55,50 +55,135 @@ function add_path {
   EXTRA_PATH=$1:${EXTRA_PATH}
 }
 
-function python-tools {
-  # Python User Env
-  PIP_USER=yes
-  export PIP_USER
-  PYTHONUSERBASE=${prebuilt}/pylocal
-  export PYTHONUSERBASE
-  add_path "${PYTHONUSERBASE}"/bin
-  pip3 install pexpect==4.8.0
-  pip3 install pytest==6.2.5
-  pip3 install pytest-repeat==0.9.1
-  pip3 install pytest-json==0.4.0
-  pip3 install pytest-ordering==0.6
-  pip3 install pyserial==3.5
-  pip3 install pyelftools
-  pip3 install cxxfilt
+function arm-clang-toolchain {
+  add_path "${prebuilt}"/clang-arm-none-eabi/bin
 
-  # MCUboot's tool for image signing and key management
-  if ! command -v imgtool &> /dev/null; then
-    pip3 install imgtool
+  if [ ! -f "${prebuilt}/clang-arm-none-eabi/bin/clang" ]; then
+    cd "${prebuilt}"
+    curl -O -L -s https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm/releases/download/release-14.0.0/LLVMEmbeddedToolchainForArm-14.0.0-linux.tar.gz
+    tar zxf LLVMEmbeddedToolchainForArm-14.0.0-linux.tar.gz
+    mv LLVMEmbeddedToolchainForArm-14.0.0 clang-arm-none-eabi
+    rm LLVMEmbeddedToolchainForArm-14.0.0-linux.tar.gz
   fi
```

Comment:
Maybe something like
```suggestion
  if [ ! -f "${prebuilt}/clang-arm-none-eabi/bin/clang" ]; then
    mkdir -p "${prebuilt}"/clang-arm-none-eabi
    curl -SL https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm/releases/download/release-14.0.0/LLVMEmbeddedToolchainForArm-14.0.0-linux.tar.gz | tar -xzC "${prebuilt}"/clang-arm-none-eabi --strip 1
  fi
```
?

Reply:
Ok. after extract the zip to the final directory, the origin method reduce to three statement, which is simple as your suggestion. That's why I keep the old flow.